EventFilter/CTPPSRawToDigi/interface/RawDataUnpacker.h

```diff
@@ -26,7 +27,14 @@ class RawDataUnpacker
     typedef uint64_t word;
 
     /// VFAT transmission modes
-    enum { vmCluster = 0x80, vmRaw = 0x90 };
+    enum { vmCluster = 0x80, vmRaw = 0x90, vmDiamondCompact = 0xB0 };
```

Comment:
Hi @davidlange6 
Sure, I will wrap this object in a `ctpps` namespace in my next iteration